CIP-0030/README.md

```diff
@@ -93,7 +93,6 @@ DataSignErrorCode {
 	ProofGeneration: 1,
 	AddressNotPK: 2,
 	UserDeclined: 3,
-	InvalidFormat: 4,
```

Comment:
This shouldn't be necessary anymore as the user only provides the payload, not the entire Sig_structure. Not providing proper hex bytes falls under the broader `APIError.InvalidRequest realm`